cmd/skywire-visor/commands/root.go

```diff
@@ -196,9 +196,7 @@ func (cfg *runCfg) runVisor() *runCfg {
 	time.Sleep(startDelay)
 
 	if cfg.conf.DmsgPty != nil {
-		err = visor.UnlinkSocketFiles(cfg.conf.AppServerSockFile, cfg.conf.DmsgPty.CLIAddr)
-	} else {
-		err = visor.UnlinkSocketFiles(cfg.conf.AppServerSockFile)
+		err = visor.UnlinkSocketFiles(cfg.conf.DmsgPty.CLIAddr)
```

Comment:
As the `else` statement is removed, the `err` variable is not assigned in all cases. E.g. if the line `185` returns an error and `cfg.conf.DmsgPty == nil` then the line `203` would be executed.